app/src/main/java/com/amaze/filemanager/utils/files/FileUtils.java

```diff
@@ -116,6 +116,26 @@ public static long folderSize(File directory, OnProgressUpdate<Long> updateState
         return length;
     }
 
+
+    public static long folderSize(File directory, final OnProgressUpdate<Long[]> updateState, Long[] spaces) {    
+        long length = 0;
+        try {
+            for (File file:directory.listFiles()) {
+                if (file.isFile()) {
+                    spaces[2] += file.length();  length += file.length();}
+                else
+                    length += folderSize(file, updateState, spaces);
```

Comment:
Lines 124-127 seem difficult to understand, is there any way to write it to be easily understandable? And are not to standards (always use brackets in `if` or `else`, except for the `else` in `else if`).

Reply:
I will add some comments

Reply:
Please make code more understandable if you can.